scripts/variantstore/scripts/import_gvs.py

```diff
@@ -378,11 +378,11 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
             any_yes=acc.any_yes | allele_YES[called_idx - 1],
             any_snp=acc.any_snp | allele_is_snp[called_idx - 1],
             any_indel=acc.any_indel | ~allele_is_snp[called_idx - 1],
-            any_snp_ok=acc.any_snp_ok | (allele_is_snp[called_idx - 1] & allele_OK[called_idx - 1]),
-            any_indel_ok=acc.any_indel_ok | (~allele_is_snp[called_idx - 1] & allele_OK[called_idx - 1]),
-        ), hl.struct(any_no=False, any_yes=False, any_snp=False, any_indel=False, any_snp_ok=False, any_indel_ok=False)))
+            all_snps_ok=acc.all_snps_ok & (~allele_is_snp[called_idx - 1] | allele_OK[called_idx - 1]),
+            all_indels_ok=acc.all_indels_ok & (allele_is_snp[called_idx - 1] | allele_OK[called_idx - 1]),
```

Comment:
I am confused as to why this is not:
```suggestion
            all_snps_ok=acc.all_snps_ok & (allele_is_snp[called_idx - 1] & allele_OK[called_idx - 1]),
            all_indels_ok=acc.all_indels_ok & (~allele_is_snp[called_idx - 1] & allele_OK[called_idx - 1]),
```
Maybe I'm just confounded by the logic

Reply:
Taking the SNP line as an example, I interpreted this as "nothing wrong SNP-wise if it's not a SNP at all, otherwise (implying it *is* a SNP) if the allele is OK.

If the allele was bad and it was an INDEL, the following line would catch it.

Reply:
Actually now that I look at this again I think I can/should clean it up a bit... I don't think we care about "any" SNPs or INDELs any more now that the default for the "ok" accumulators is `True`.